src/objects/canvasManager.js

```diff
@@ -0,0 +1,36 @@
+
+/**
+ * This class will expose the p5 canvas, which we can use to set mouseclick listeners, as well as
+ * properties involving the width and height of the canvas.
+ */
+class CanvasManager {
+
+    // Dimensions of the canvas in pixels
+    // TODO: Set these as the height and width of the window on load, and change them whenever the window is resized.
+    /**
+     * The width of the canvas.
+     */
+    width = 640;
```

Comment:
These comments make it so that when you hover over these properties in an IDE like VS Code, you can see some potentially helpful information.